File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/CompleteStagedRollout.java
package com.dipien.google.play.api.task;

import com.dipien.google.play.api.App;
import com.dipien.google.play.api.PublishingService;

public class CompleteStagedRollout extends BaseTask {
	
	public CompleteStagedRollout() {
		setDescription("Rollout the release to 100% of users");
	}

	@Override
	protected void onExecute(App app) {
		new PublishingService().completeStagedRollout(app);
	}

}

File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/CompleteStagedRollout.kt
package com.dipien.google.play.api.task

import com.dipien.google.play.api.App
import com.dipien.google.play.api.PublishingService

class CompleteStagedRollout : BaseTask() {
    override fun onExecute(app: App) {
        PublishingService().completeStagedRollout(app)
    }

    init {
        description = "Rollout the release to 100% of users"
    }
}
